src/Platforms/SQLitePlatform.php

```diff
@@ -772,6 +772,7 @@ private function getSimpleAlterTableSQL(TableDiff $diff): array|false
 
             $type = $definition['type'];
 
+            /** @psalm-suppress RiskyTruthyFalsyComparison */
```

Comment:
I'd like to improve this code in 4.1 which should fix the error. This suppression is hopefully only temporary. 🤞🏻